Train.json

```diff
@@ -1498,6 +1538,20 @@
 			"cost": 380000,
 			"operationCosts": 120,
 			"equipments": ["FR","LU"]
+		},{
+			"id": 40100,
+			"group": 0,
+			"name": "SNCF CC 40100 Nez Cassé",
+			"shortcut": "CC 40100",
+			"speed": 240,
+			"weight": 110,
+			"force": 210,
+			"length": 22,
+			"drive": 1,
+			"reliability": 0.95,
+			"cost": 550000,
+			"operationCosts": 120,
+			"equipments": ["FR","TVM", "CH", "BE", "LU", "NL", "DE", "AT", "IT"]
```

Comment:
Die Lok hatte eher keine TVM.

Reply:
Ich hatte gestern ein Foto gesehen wo eine Nez Cassé (sollte eine 40100 gewesen sein) nachts auf einer (anscheinen Hochgeschwindigkeits-) Strecke gefahren ist. Bildunterschrift war sowas wie "fuhr 240 km/h mit Nachtzug über TGV Strecke". So weit meine Erinnerung - jetzt finde ich das natürlich nicht wieder. Könnte natürlich sein, daß nur eine/wenige Loks damit nachträglich ausgerüstet wurde - oder ich das falsch in Erinnerung habe.

Reply:
Es ist in Frankreich nicht vorgesehen mit lokbespannten Zügen über LGV zu fahren. Einzige Ausnahme ist die LGV Bretagne-Pays de la Loire wo theoretisch sogar Güterzüge fahren können.

Zusätzliches Problem: Die französischen LGV sind zwischen 23:00 Uhr und 5:30 grundsätzlich für Bauarbeiten gesperrt.